populator.go

```diff
@@ -87,7 +87,8 @@ func (p *Populator) generateInsertStmt(fixture Fixture, data []map[string]interf
 		args = append(args, recordArgs...)
 	}
 	values := strings.Join(placeholders, ",")
-	return fmt.Sprintf(query, fixture.TableName, strings.Join(keys, ","), values), args
+	keysWithQuotes := surroundKeysWithQuotes(keys, p.Driver)
```

Comment:
You are modifying the slice in place here, so no need to use the return value,
and no need to return anything from the function either.